web/src/admin/stages/redirect/RedirectStageForm.ts

```diff
@@ -0,0 +1,153 @@
+import { RenderFlowOption } from "@goauthentik/admin/flows/utils";
+import { BaseStageForm } from "@goauthentik/admin/stages/BaseStageForm";
+import { DEFAULT_CONFIG } from "@goauthentik/common/api/config";
+import "@goauthentik/elements/forms/HorizontalFormElement";
+
+import { msg } from "@lit/localize";
+import { TemplateResult, html } from "lit";
+import { customElement, property } from "lit/decorators.js";
+
+import {
+    Flow,
+    FlowsApi,
+    FlowsInstancesListRequest,
+    RedirectStage,
+    RedirectStageModeEnum,
+    StagesApi,
+} from "@goauthentik/api";
+
+@customElement("ak-stage-redirect-form")
+export class RedirectStageForm extends BaseStageForm<RedirectStage> {
+    @property({ type: String })
+    mode: string = RedirectStageModeEnum.Static;
+
+    loadInstance(pk: string): Promise<RedirectStage> {
+        return new StagesApi(DEFAULT_CONFIG)
+            .stagesRedirectRetrieve({
+                stageUuid: pk,
+            })
+            .then((stage) => {
+                this.mode = stage.mode ?? RedirectStageModeEnum.Static;
+                return stage;
+            });
+    }
+
+    async send(data: RedirectStage): Promise<RedirectStage> {
+        if (this.instance) {
+            return new StagesApi(DEFAULT_CONFIG).stagesRedirectUpdate({
+                stageUuid: this.instance.pk || "",
+                redirectStageRequest: data,
+            });
+        } else {
+            return new StagesApi(DEFAULT_CONFIG).stagesRedirectCreate({
+                redirectStageRequest: data,
+            });
+        }
+    }
+
+    renderForm(): TemplateResult {
+        return html`<span>
+                ${msg("Redirect the user to another flow, potentially with all gathered context")}
+            </span>
+            <ak-form-element-horizontal label=${msg("Name")} required name="name">
+                <input
+                    type="text"
+                    value="${this.instance?.name ?? ""}"
+                    class="pf-c-form-control"
+                    required
+                />
+            </ak-form-element-horizontal>
+            <ak-form-group .expanded=${true}>
```

Comment:
Suggestion: Just `expanded`, please.  You don't need the whole `.expanded=${true}`.  Really, you don't.